Release fails due to Checkstyle failure in Maven plugin generated-sources HelpMojo.java

#508 release (for #509) of v2.5.2 failed (partially or fully, unclear?) due to this:

```
    [ERROR] /home/vorburger/dev/MariaDB4j/target/checkout/mariaDB4j-maven-plugin/target/generated-sources/plugin/ch/vorburger/mariadb4j/HelpMojo.java:257:5: '{' at column 5 should be on the previous line. [LeftCurly]    [ERROR] /home/vorburger/dev/MariaDB4j/target/checkout/mariaDB4j-maven-plugin/target/generated-sources/plugin/ch/vorburger/mariadb4j/HelpMojo.java:259:9: Distance between variable 'parameterDescription' declaration and its first usage is 5, but allowed 3.  Consider making that variable final if you still need to store its value in advance (before method calls that might have side effects on the original value). [VariableDeclarationUsageDistance]    [ERROR] /home/vorburger/dev/MariaDB4j/target/checkout/mariaDB4j-maven-plugin/target/generated-sources/plugin/ch/vorburger/mariadb4j/HelpMojo.java:263:9: '{' at column 9 should be on the previous line. [LeftCurly]    [ERROR] /home/vorburger/dev/MariaDB4j/target/checkout/mariaDB4j-maven-plugin/target/generated-sources/plugin/ch/vorburger/mariadb4j/HelpMojo.java:264:11: 'if' child has incorrect indentation level 10, expected level should be 12. [Indentation]    [ERROR] /home/vorburger/dev/MariaDB4j/target/checkout/mariaDB4j-maven-plugin/target/generated-sources/plugin/ch/vorburger/mariadb4j/HelpMojo.java:269:9: '{' at column 9 should be on the previous line. [LeftCurly]    [ERROR] /home/vorburger/dev/MariaDB4j/target/checkout/mariaDB4j-maven-plugin/target/generated-sources/plugin/ch/vorburger/mariadb4j/HelpMojo.java:275:9: '{' at column 9 should be on the previous line. [LeftCurly]    [ERROR] /home/vorburger/dev/MariaDB4j/target/checkout/mariaDB4j-maven-plugin/target/generated-sources/plugin/ch/vorburger/mariadb4j/HelpMojo.java:281:9: '{' at column 9 should be on the previous line. [LeftCurly]    [ERROR] /home/vorburger/dev/MariaDB4j/target/checkout/mariaDB4j-maven-plugin/target/generated-sources/plugin/ch/vorburger/mariadb4j/HelpMojo.java:285:9: '{' at column 9 should be on the previous line. [LeftCurly]    [ERROR] /home/vorburger/dev/MariaDB4j/target/checkout/mariaDB4j-maven-plugin/target/generated-sources/plugin/ch/vorburger/mariadb4j/HelpMojo.java:303:5: '{' at column 5 should be on the previous line. [LeftCurly]    [ERROR] /home/vorburger/dev/MariaDB4j/target/checkout/mariaDB4j-maven-plugin/target/generated-sources/plugin/ch/vorburger/mariadb4j/HelpMojo.java:307:9: '{' at column 9 should be on the previous line. [LeftCurly]    [ERROR] /home/vorburger/dev/MariaDB4j/target/checkout/mariaDB4j-maven-plugin/target/generated-sources/plugin/ch/vorburger/mariadb4j/HelpMojo.java:323:5: '{' at column 5 should be on the previous line. [LeftCurly]    [ERROR] /home/vorburger/dev/MariaDB4j/target/checkout/mariaDB4j-maven-plugin/target/generated-sources/plugin/ch/vorburger/mariadb4j/HelpMojo.java:325:9: '{' at column 9 should be on the previous line. [LeftCurly]    [ERROR] /home/vorburger/dev/MariaDB4j/target/checkout/mariaDB4j-maven-plugin/target/generated-sources/plugin/ch/vorburger/mariadb4j/HelpMojo.java:341:5: '{' at column 5 should be on the previous line. [LeftCurly]    [ERROR] /home/vorburger/dev/MariaDB4j/target/checkout/mariaDB4j-maven-plugin/target/generated-sources/plugin/ch/vorburger/mariadb4j/HelpMojo.java:349:9: '{' at column 9 should be on the previous line. [LeftCurly]    [ERROR] /home/vorburger/dev/MariaDB4j/target/checkout/mariaDB4j-maven-plugin/target/generated-sources/plugin/ch/vorburger/mariadb4j/HelpMojo.java:365:5: '{' at column 5 should be on the previous line. [LeftCurly]    [ERROR] /home/vorburger/dev/MariaDB4j/target/checkout/mariaDB4j-maven-plugin/target/generated-sources/plugin/ch/vorburger/mariadb4j/HelpMojo.java:372:9: '{' at column 9 should be on the previous line. [LeftCurly]    [ERROR] /home/vorburger/dev/MariaDB4j/target/checkout/mariaDB4j-maven-plugin/target/generated-sources/plugin/ch/vorburger/mariadb4j/HelpMojo.java:374:13: '{' at column 13 should be on the previous line. [LeftCurly]    [ERROR] /home/vorburger/dev/MariaDB4j/target/checkout/mariaDB4j-maven-plugin/target/generated-sources/plugin/ch/vorburger/mariadb4j/HelpMojo.java:376:17: '{' at column 17 should be on the previous line. [LeftCurly]    [ERROR] /home/vorburger/dev/MariaDB4j/target/checkout/mariaDB4j-maven-plugin/target/generated-sources/plugin/ch/vorburger/mariadb4j/HelpMojo.java:382:17: '{' at column 17 should be on the previous line. [LeftCurly]    [ERROR] /home/vorburger/dev/MariaDB4j/target/checkout/mariaDB4j-maven-plugin/target/generated-sources/plugin/ch/vorburger/mariadb4j/HelpMojo.java:388:13: '{' at column 13 should be on the previous line. [LeftCurly]    [ERROR] /home/vorburger/dev/MariaDB4j/target/checkout/mariaDB4j-maven-plugin/target/generated-sources/plugin/ch/vorburger/mariadb4j/HelpMojo.java:391:17: '{' at column 17 should be on the previous line. [LeftCurly]    [ERROR] /home/vorburger/dev/MariaDB4j/target/checkout/mariaDB4j-maven-plugin/target/generated-sources/plugin/ch/vorburger/mariadb4j/HelpMojo.java:395:17: '{' at column 17 should be on the previous line. [LeftCurly]    [ERROR] /home/vorburger/dev/MariaDB4j/target/checkout/mariaDB4j-maven-plugin/target/generated-sources/plugin/ch/vorburger/mariadb4j/HelpMojo.java:399:17: '{' at column 17 should be on the previous line. [LeftCurly]    [ERROR] /home/vorburger/dev/MariaDB4j/target/checkout/mariaDB4j-maven-plugin/target/generated-sources/plugin/ch/vorburger/mariadb4j/HelpMojo.java:414:5: '{' at column 5 should be on the previous line. [LeftCurly]    [ERROR] /home/vorburger/dev/MariaDB4j/target/checkout/mariaDB4j-maven-plugin/target/generated-sources/plugin/ch/vorburger/mariadb4j/HelpMojo.java:417:9: '{' at column 9 should be on the previous line. [LeftCurly]    [ERROR] /home/vorburger/dev/MariaDB4j/target/checkout/mariaDB4j-maven-plugin/target/generated-sources/plugin/ch/vorburger/mariadb4j/HelpMojo.java:421:9: '{' at column 9 should be on the previous line. [LeftCurly]    [ERROR] /home/vorburger/dev/MariaDB4j/target/checkout/mariaDB4j-maven-plugin/target/generated-sources/plugin/ch/vorburger/mariadb4j/HelpMojo.java:423:13: '{' at column 13 should be on the previous line. [LeftCurly]    [ERROR] /home/vorburger/dev/MariaDB4j/target/checkout/mariaDB4j-maven-plugin/target/generated-sources/plugin/ch/vorburger/mariadb4j/HelpMojo.java:432:5: '{' at column 5 should be on the previous line. [LeftCurly]    [ERROR] /home/vorburger/dev/MariaDB4j/target/checkout/mariaDB4j-maven-plugin/target/generated-sources/plugin/ch/vorburger/mariadb4j/HelpMojo.java:435:9: '{' at column 9 should be on the previous line. [LeftCurly]    Audit done.    [INFO] ------------------------------------------------------------------------    [INFO] Reactor Summary for mariaDB4j-pom 2.5.2:    [INFO]    [INFO] mariaDB4j-pom ...................................... SUCCESS [  4.821 s]    [INFO] mariaDB4j-core ..................................... SUCCESS [ 12.540 s]    [INFO] mariaDB4j (all-in-one artifact) .................... SUCCESS [01:06 min]    [INFO] mariaDB4j-springboot ............................... SUCCESS [ 12.754 s]    [INFO] mariaDB4j-app ...................................... SUCCESS [ 18.247 s]    [INFO] mariaDB4j-maven-plugin ............................. FAILURE [02:46 min]    [INFO] ------------------------------------------------------------------------    [INFO] BUILD FAILURE    [INFO] ------------------------------------------------------------------------    [INFO] Total time:  04:41 min    [INFO] Finished at: 2021-12-11T14:50:14+01:00    [INFO] ------------------------------------------------------------------------    [WARNING] The requested profile "release" could not be activated because it does not exist.    [ERROR] Failed to execute goal org.apache.maven.plugins:maven-checkstyle-plugin:3.1.2:check (validate) on project mariaDB4j-maven-plugin: Failed during checkstyle execution: There are 72 errors reported by Checkstyle 9.2 with checkstyle.xml ruleset. -> [Help 1]    [ERROR]    [ERROR] To see the full stack trace of the errors, re-run Maven with the -e switch.    [ERROR] Re-run Maven using the -X switch to enable full debug logging.    [ERROR]    [ERROR] For more information about the errors and possible solutions, please read the following articles:    [ERROR] [Help 1] http://cwiki.apache.org/confluence/display/MAVEN/MojoExecutionException    [ERROR]    [ERROR] After correcting the problems, you can resume the build with the command    [ERROR]   mvn <args> -rf :mariaDB4j-maven-plugin[INFO] ------------------------------------------------------------------------[INFO] Reactor Summary for mariaDB4j-pom 2.5.3-SNAPSHOT:[INFO][INFO] mariaDB4j-pom ...................................... FAILURE [05:17 min][INFO] mariaDB4j-core ..................................... SKIPPED[INFO] mariaDB4j (all-in-one artifact) .................... SKIPPED[INFO] mariaDB4j-springboot ............................... SKIPPED[INFO] mariaDB4j-app ...................................... SKIPPED[INFO] mariaDB4j-maven-plugin ............................. SKIPPED[INFO] ------------------------------------------------------------------------[INFO] BUILD FAILURE[INFO] ------------------------------------------------------------------------[INFO] Total time:  05:17 min[INFO] Finished at: 2021-12-11T14:50:14+01:00[INFO] ------------------------------------------------------------------------[ERROR] Failed to execute goal org.apache.maven.plugins:maven-release-plugin:3.0.0-M4:perform (default-cli) on project mariaDB4j-pom: Maven execution failed, exit code: '1' -> [Help 1][ERROR]
```

--- a/mariaDB4j-maven-plugin/pom.xml
+++ b/mariaDB4j-maven-plugin/pom.xml
@@ -207,18 +207,6 @@
                 </executions>
             </plugin>
             <plugin>
-                <artifactId>maven-source-plugin</artifactId>
-                <executions>
-                    <execution>
-                        <id>attach-sources</id>
-                        <phase>deploy</phase>
-                        <goals>
-                            <goal>jar-no-fork</goal>
-                        </goals>
-                    </execution>
-                </executions>
-            </plugin>
-            <plugin>
                 <!-- Include slf4j-api in the plugin jar to ensure some versions of maven don't have logging issues.
                      Deprecate when min maven version is 3.3.9+ currently we allow 3.1.1+
                      see https://issues.apache.org/jira/browse/MNG-5787 for details.

--- a/pom.xml
+++ b/pom.xml
@@ -114,6 +114,8 @@
                 </dependencies>
 				<configuration>
 					<configLocation>checkstyle.xml</configLocation>
+					<!-- https://github.com/vorburger/MariaDB4j/issues/511 -->
+					<sourceDirectories>${project.build.sourceDirectory}</sourceDirectories>
 					<includeTestSourceDirectory>true</includeTestSourceDirectory>
 					<encoding>UTF-8</encoding>
 					<consoleOutput>true</consoleOutput>

